frontend/components/forms/fields/Checkbox/Checkbox.tsx

```diff
@@ -118,7 +118,7 @@ const Checkbox = (props: ICheckboxProps) => {
           type="checkbox"
           ref={inputRef}
           name={name}
-          checked={value || undefined}
+          checked={!!value}
```

Comment:
Without this change I get a warning from React about moving from an uncontrolled to a controlled input when the checkbox was checked, which makes sense since we're going from `undefined` to `true`.  I've been unable to figure out why we were getting this warning previously though.